Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ccl/configuration/update/CCLSettings.kt

```diff
@@ -36,6 +36,9 @@ class CCLSettings @Inject constructor(
             }
         }
 
+    val admissionScenarioId = dataStoreFlow
```

Comment:
Maybe we should refer to it as `selectedAdmissionScenarioId` in this class to make it more clear?

Reply:
I think the `selection` is more related to the `UI` in the settings `admissionScenarioId` is fine

Reply:
Okay.